libs/langchain/langchain/utilities/clickup.py

```diff
@@ -166,7 +149,29 @@ def query_tasks(self, query: str) -> str:
 
         data = response.json()
         return data
+    
+    def update_task(self, query: str) -> str:
+        """
+            Update an attribute of a specified task
+        """
+        task = self.get_task(query)
+        
+        params = json.loads(query)
+        url = "https://api.clickup.com/api/v2/task/" + params['task_id']
+
+        query = {
+            "custom_task_ids": "true",
+            "team_id": self.team_id,
```

Comment:
I'll update this in my PR to get this data in the validation function so that we don't need to hard code it in anywhere